interns-react-weekly-assignments/.eslintrc.json

```diff
@@ -22,6 +22,7 @@
     ],
     "rules": {
         // "no-duplicate-imports": "error"
+        "eol-last":["error","always"]
```

Comment:
Please rebase to main and resolve conflicts to not have other changes in you PR.